packages/cli/lib/commands/create.js

```diff
@@ -1,12 +1,10 @@
 const ora = require('ora');
-const glob = require('glob');
+const glob = require('tiny-glob');
 const gittar = require('gittar');
-const fs = require('fs.promised');
-const copy = require('ncp');
-const { green } = require('chalk');
+const fs = require('../fs');
+const { green } = require('kleur');
 const { resolve, join } = require('path');
 const { prompt } = require('inquirer');
```

Comment:
I would vote for `prompts` here – waaay smaller but up to you

Reply:
I don't mind